CRVReco/src/CrvRecoPulsesFinder_module.cc

```diff
@@ -154,6 +164,9 @@ namespace mu2e
       double calibPulseArea = calib.pulseArea(channel);
       double calibPulseHeight = calib.pulseHeight(channel);
       double timeOffset = calib.timeOffset(channel);
+      timeOffset*=_timeOffsetScale;   //random time offsets can be scaled to a wider or smaller spread
+      if(timeOffset<_timeOffsetCutoffLow)  timeOffset=_timeOffsetCutoffLow;  //random time offsets can be cutoff at some limit
+      if(timeOffset>_timeOffsetCutoffHigh) timeOffset=_timeOffsetCutoffHigh;
```

Comment:
This will produce a time distribution with peaks at the two limits.  Is that the intention?

Reply:
The intention is to prevent extremely small or large numbers that may be present in randomly generated DB tables. The tables should be generated (and the scale factor should be set) in such a way that this situation either never happens or only in very rare occasions.

Reply:
What will you do with out-oft-time real data?  If you will discard it, you should do the same here.  Or do I misunderstand the scenario?

Reply:
These cutoffs are meant to be applied only to randomly generated time offsets (where a small fraction of them may randomly end up unrealistically far away from 0). Measured (real) time offsets should not be cut off. Therefore, the cutoffs for measured time offsets (i.e. not randomly generated) need to be set to values so that none of the time offsets are cut.
I introduced these cutoffs and scale factors so that one can test different situations without changing the DB table.

Reply:
Thanks for fixing the clang tidy issues and for the explanations.  The cutoff sound like they will need adjustments when they encounter real data - many things will.